core/pom.rb

```diff
@@ -65,7 +65,7 @@
   jar 'org.osgi:org.osgi.core:5.0.0', :scope => 'provided'
 
   # joda timezone must be before joda-time to be packed correctly
-  jar 'org.jruby:joda-timezones:${tzdata.version}', :scope => '${tzdata.scope}'
+  # jar 'org.jruby:joda-timezones:${tzdata.version}', :scope => '${tzdata.scope}'
```

Comment:
I believe this is redundant given the joda-time includes the up-to-date tz-data and we want to make sure to use those